generator/generator_tool/generator_tool.cpp

```diff
@@ -69,8 +69,9 @@ DEFINE_bool(split_by_polygons, false,
 // Routing.
 DEFINE_string(osrm_file_name, "", "Input osrm file to generate routing info.");
 DEFINE_bool(make_routing, false, "Make routing info based on osrm file.");
-DEFINE_bool(make_cross_section, false, "Make cross section in routing file for cross mwm routing.");
+DEFINE_bool(make_cross_section, false, "Make cross section in routing file for cross mwm routing (for old OSRM routing).");
 DEFINE_bool(make_routing_index, false, "Make sections with the routing information.");
+DEFINE_bool(make_cross_mwm, false, "Make section for cross mwm routing (for new AStar routing).");
```

Comment:
Actually AStar algorithm is a Dijkstra algorithm, there's no real difference. But agree, I don't see why do we need to mention exact algorithm name in these flags.